loops-and-arrays.js

```diff
@@ -29,7 +30,14 @@ const numbers = [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
 
-
+for (let i = 0; i < numbers.length; i++){
+    // console.log(numbers[i]);
+    if ( numbers[i] % 2 === 0) {
+        console.log(numbers[i] * 2);
+    } else {
+        console.log(numbers[i] * 3);
+    }
+}
 
```

Comment:
Deze ziet er ook goed uit.  Antwoordmodel is net iets anders maar dit werkt natuurlijk ook!